### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by KonradStaniec**
Simple docs to cover:
- build instructions
- commands available to the user

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,80 @@
 # CLI-TOOLS
```
Comment (vitsalis) [inline README.md:1]:
```suggestion
# CLI-Tools
```

kinda shouty? 😆

File: README.md [hunk 2]
```diff
@@ -1,3 +1,80 @@
...
 
-Set of CLI tools to run as Batch jobs on phase-1 main net
+Set of CLI tool which are used for different purposes. See  [deployment guide](/docs/commands.md)
```
Comment (vitsalis) [inline README.md:3]:
```suggestion
Set of CLI tools which are used for different purposes. See  [deployment guide](/docs/commands.md)
```

Maybe let's say more about the "different purposes"?

File: README.md [hunk 3]
```diff
@@ -1,3 +1,80 @@
...
+to review all available commands
```
Comment (vitsalis) [inline README.md:4]:
```suggestion
to review all available commands.
```

File: README.md [hunk 4]
```diff
@@ -1,3 +1,80 @@
...
+
+### Installation
+
+#### Prerequisites
+
+This project requires Go version 1.21 or later.
+
+Install Go by following the instructions on the official Go installation guide.
```
Comment (vitsalis) [inline README.md:12]: Link to the installation guide?

File: README.md [hunk 5]
```diff
@@ -1,3 +1,80 @@
...
+#### Build and install the binary
+
+At the top-level directory of the project
+
+```shell
+make install
+```
+
+The above command will build and install the `cli-tools` binary to
+`$GOPATH/bin`, which is the cli program with set of different commands.
```
Comment (vitsalis) [inline README.md:36]:
```suggestion
`$GOPATH/bin`.
```
No need for this I think

File: docs/commands.md [hunk 6]
```diff
@@ -138,3 +138,51 @@ $ bitcoin-cli -rpcwallet=timestamp-w gettransaction 25b65b31c6f4d2f46ebeb5fa4c9a
...
+
+All 3 commands:
+- `create-phase1-staking-tx`
+- `create-phase1-unbonding-request`
+- `create-phase1-withdaw-request`
+
+Are used to create transactions that should be compatible with phase-1 system.
+The main goal of those commands is to ease up testing of phase-1 system from the command
+line.
+
```
Comment (vitsalis) [inline docs/commands.md:152]:
```suggestion
The following set of commands are used to create phase-1 compatible transactions
with the goal to ease up testing of the phase-1 system:
- `create-phase1-staking-tx`
- `create-phase1-unbonding-request`
- `create-phase1-withdaw-request`
```

File: docs/commands.md [hunk 7]
```diff
@@ -138,3 +138,51 @@ $ bitcoin-cli -rpcwallet=timestamp-w gettransaction 25b65b31c6f4d2f46ebeb5fa4c9a
...
+Disclaimer: Those commands should only be used for testing purposes and should not be
+used with real BTC.
+
+## Dump config command
+
+Some of the commands require a config file to work properly. To generate a config
+file `dump-cfg` command can be used.
```
Comment (vitsalis) [inline docs/commands.md:159]:
```suggestion
file the `dump-cfg` command can be used.
```

File: docs/commands.md [hunk 8]
```diff
@@ -138,3 +138,51 @@ $ bitcoin-cli -rpcwallet=timestamp-w gettransaction 25b65b31c6f4d2f46ebeb5fa4c9a
...
+
+Command:
+
```
Comment (vitsalis) [inline docs/commands.md:162]:
```suggestion
```

File: docs/commands.md [hunk 9]
```diff
@@ -138,3 +138,51 @@ $ bitcoin-cli -rpcwallet=timestamp-w gettransaction 25b65b31c6f4d2f46ebeb5fa4c9a
...
+```shell
+cli-tools dump-cfg --config "./config.toml"
+```
+
+will generate `config.toml` file in the same directory as `cli-tools` program
+binary.
```
Comment (vitsalis) [inline docs/commands.md:168]:
```suggestion
will generate a `config.toml` file in the same directory as the one 
the `cli-tools` program binary resides in.
```

File: docs/commands.md [hunk 10]
```diff
@@ -138,3 +138,51 @@ $ bitcoin-cli -rpcwallet=timestamp-w gettransaction 25b65b31c6f4d2f46ebeb5fa4c9a
...
+- global parameters paths
+
+Example:
+
+```shell
+cli-tools run-unbonding-pipeline --config "./config/config.toml" --params "./config/global-params.json"
+```
+
+For these commands to work, they must have access to the data source (mongo-db) containing
+unbonding transactions already signed by staker, and validated by some other process
```
Comment (vitsalis) [inline docs/commands.md:188]:
```suggestion
unbonding transactions already signed by staker and validated by some other process.
```

**Review by vitsalis — APPROVED** [review]
Nice! Suggestions:

File: README.md (hunk 1 above)
Comment (KonradStaniec) [reply README.md:1]: hahah now that you mention it, I cannot unsee readme shouting at me 😅 sure will change this!

**Pull request merged**